modelconverter/utils/onnx_tools.py

```diff
@@ -1,10 +1,15 @@
 import logging
+from dataclasses import dataclass
 from pathlib import Path
-from typing import Dict
+from typing import Dict, List, Optional, Tuple
 
+import numpy as np
 import onnx
+import onnx_graphsurgeon as gs
```

Comment:
Could we use `graphsurgeon` to also simplify the code in `onnx_attach_normalization_to_inputs`? Or add it to the `ONNXModifier`

Reply:
Even though I would like the `onnx_attach_normalization_to_inputs` to be another function of `ONNXModifier`, I think its better to leave this one outside of the class. The reason is that its the fallback solution in case the optimisations fail or do not produce the correct results compared to the "base" model which is produced by this exact function.

Reply:
But the `onnx_attach_normalization_to_inputs` alone can be changed to use `graphsurgeon` instead, yeah.